tool/tctl/common/auth_command.go

```diff
@@ -123,6 +124,8 @@ func (a *AuthCommand) Initialize(app *kingpin.Application, _ *tctlcfg.GlobalCLIF
 		fmt.Sprintf("export certificate type (%v)", strings.Join(allowedCertificateTypes, ", "))).
 		EnumVar(&a.authType, allowedCertificateTypes...)
 	a.authExport.Flag("integration", "Name of the integration. Only applies to \"github\" CAs.").StringVar(&a.integration)
+	a.authExport.Flag("out-prefix", "If set writes exported authorities to files with the given prefix").
```

Comment:
nit: may want to mention "path" as well as prefix.

Reply:
Done.